upf/upf_pfcp.c

```diff
@@ -2575,10 +2575,9 @@ process_urrs (vlib_main_t * vm, upf_session_t * sess,
 
   if (PREDICT_FALSE (status != URR_OK))
     {
-      vec_validate_ha (uev, 0, sizeof (upf_event_urr_hdr_t), 0);
-      ueh =
-	(upf_event_urr_hdr_t *) vec_header (uev,
-					    sizeof (upf_event_urr_hdr_t));
+      /* TBD: re-verify! */
```

Comment:
`vec_validate_hap` change looks valid, header size need to be provided in case of empty vector. What to verify?